InfoLogger/test/lib/mocha-sqldatasource.js

```diff
@@ -0,0 +1,221 @@
+const assert = require('assert');
+const sinon = require('sinon');
+const SQLDataSource = require('../../lib/SQLDataSource.js');
+const {MySQL} = require('@aliceo2/web-ui');
+
+
+describe('SQLDataSource', () => {
+  const config = {
+    mysql: {
+      host: 'localhost',
+      user: 'root',
+      password: 'admintest',
+      database: 'INFOLOGGER',
+      port: 8909
```

Comment:
I have DB running at different port, maybe let's just have MySQL checks only at CI? This is already the case for https://github.com/AliceO2Group/WebUi/blob/dev/Framework/Backend/test/mocha-db.js

Reply:
These are just unit tests and will not make use of any mysql instances. The configuration is only used to mock creating an instance of mysql driver

Reply:
Ah okay, maybe just use values from `config-default.js`?

Reply:
Updated to use `config-default.js`